.azure-pipelines/linux_build.yml

```diff
@@ -9,7 +9,7 @@ steps:
     conda create --name rdkit_build $(python) cmake \
         boost-cpp=$(boost_version) \
         py-boost=$(boost_version) \
-        numpy pillow eigen pandas matplotlib-base \
+        numpy pillow eigen pandas=2.0 matplotlib-base=3.7 \
```

Comment:
You have pandas 2.1 everywhere else, is this intentional?

Reply:
> You have pandas 2.1 everywhere else, is this intentional?

yeah, 2.1 isn't available for python 3.7